### ISSUE TITLE
`tag:-issue` not filtering as expected

### ISSUE THREAD

**Issue by unitof** [yescomment/bh-flaminghydra-ghosttheme#28]
Hey @norberthnd,

Our post `/preview` (id `654a84ade22f6200013f598f`) is tagged `issue`, but still showing up in author pages.

Example: https://flaminghydra.com/contributor/tom-scocca/

Let me know if I'm missing anything obvious.

**Comment by norberthnd** [comment]
@unitof Yes, filters are based on the slug not the tag name.
In this case the slug is 'issues'.

![CleanShot 2024-01-27 at 08 33 33@2x](https://github.com/yescomment/bh-flaminghydra-ghosttheme/assets/13472960/feea563c-ddb8-475e-a00e-ac0713d0ec1b)

Two options:
- change the code to use 'issues' in filters
- change the slug from `issues` to `issue`

I did the second as it's quicker, but later it can be changed. Now this post is not showing up anymore.
I saw you commented out the 'Appears in' section.
